man-roxygen/pwidth_pheight.R

```diff
@@ -1,3 +1,5 @@
+#' @param pwidth Default width of plots printed to files in units of
+#' \code{punits}. The default is \code{pwidth=6.5}.
```

Comment:
Just curious why you used `\code{<text>}` rather than `punits`? I am not recommending any changes just curious.

Reply:
I just copied and pasted from an old definition and didn't convert it over to markdown syntax. I can make that change, though, or maybe open an new issue to check for instances of \code{} throughout r4ss and change them to backticks (I think that is recommended, correct)?

Reply:
I changed them all to backticks here